README.md

```diff
@@ -43,7 +46,7 @@ Homebrew is a member of the [Software Freedom Conservancy](http://sfconservancy.
   - Make check payable to "Software Freedom Conservancy, Inc." and place "Directed donation: Homebrew" in the memo field.  Checks should then be mailed to:
     - Software Freedom Conservancy, Inc.
       137 Montague ST  STE 380
-      BROOKLYN, NY 11201             USA
+      BROOKLYN, NY 11201 USA
```

Comment:
I don't think the address renders as it should, neither before your change nor now. Would you mind looking into how this can be improved? I think we could add some delimiter between what are individual lines here (in the Markdown) or we could change the markup so these actually render as individual lines in GitHub's rendering of the README.